Notebook parte 1

In [1]:
pip install pandas

Note: you may need to restart the kernel to use updated packages.



[notice] A new release of pip is available: 23.2.1 -> 26.1.2
[notice] To update, run: C:\Users\Axel\AppData\Local\Microsoft\WindowsApps\PythonSoftwareFoundation.Python.3.11_qbz5n2kfra8p0\python.exe -m pip install --upgrade pip


In [1]:
import pandas as pd

In [2]:
import pandas as pd

df_original = pd.read_csv('oferta_gastronomica.csv', sep=';', encoding='latin-1', on_bad_lines='skip')

In [ ]:
df_original.head(5)

,long,lat,id,nombre,categoria,cocina,ambientacion,telefono,mail,horario,calle_nombre,calle_altura,calle_cruce,direccion_completa,barrio,comuna,codigo_postal,codigo_postal_argentino
0,"-58,36330675","-34,62296347",1602,GUANTANAMERA,RESTAURANTE,CUBANA,MODERNO,4314 0770,NaN,NaN,MOREAU DE JUSTO ALICIA,2092,NaN,"MOREAU DE JUSTO, ALICIA AV. 2092",Puerto Madero,1,1107.0,C1107AFP
1,"-58,39232963","-34,62155353",676,1843 VINACOTECA,VINERIA,NaN,NaN,4779 0806,NaN,lunes a domingo de 8 a 00,HUMBERTO 1�,1843,NaN,HUMBERTO 1� 1843,San Cristobal,3,NaN,NaN
2,"-58,45354227","-34,56031135",487,2080 EXPRESS,RESTAURANTE,TRADICIONAL,CLASICO PORTE�O,4788 4500,NaN,lunes a domingo de 8 a 00,JURAMENTO,2080,NaN,JURAMENTO AV. 2080,Belgrano,13,1428.0,C1428DNH
3,"-58,47549105","-34,6770507",2737,22,CAFE,MINUTAS,MODERNO,NaN,NaN,NaN,MURGUIONDO,4188,NaN,MURGUIONDO 4188,Villa Lugano,8,1439.0,C1439FTB
4,"-58,4254372","-34,58569501",2,4 EN 2 I,RESTAURANTE,NaN,NaN,4832 9510,NaN,NaN,BORGES JORGE LUIS,2095,NaN,"BORGES, JORGE LUIS 2095",Palermo,14,1425.0,C1425FFA


In [ ]:
print(df_original)
print(df_original.dtypes)

              long           lat    id            nombre    categoria  \
0     -58,36330675  -34,62296347  1602      GUANTANAMERA  RESTAURANTE   
1     -58,39232963  -34,62155353   676   1843 VINACOTECA      VINERIA   
2     -58,45354227  -34,56031135   487      2080 EXPRESS  RESTAURANTE   
3     -58,47549105   -34,6770507  2737                22         CAFE   
4      -58,4254372  -34,58569501     2          4 EN 2 I  RESTAURANTE   
...            ...           ...   ...               ...          ...   
2818  -58,40110747   -34,5996429   893  ZOE PIZZA & CAFE  RESTAURANTE   
2819  -58,38637192  -34,60717761  2546             ZOMBI  RESTAURANTE   
2820  -58,46310827  -34,54336919  1521              ZONA  RESTAURANTE   
2821  -58,39569675  -34,60221271   894           ZOO BAR  RESTAURANTE   
2822  -58,43986319  -34,60086815  2854          Angelito          BAR   

             cocina          ambientacion   telefono               mail  \
0            CUBANA               MODERNO  4314 

In [ ]:
#CANTIDAD NULOS 

print(df_original.isnull().sum())


print("-" * 30)

print("Porcentaje de nulos")

porcentaje = (df_original.isnull().sum() / len(df_original) * 100).round(1)
print(porcentaje)

long                          0
lat                           0
id                            0
nombre                        0
categoria                   116
cocina                      359
ambientacion                559
telefono                    520
mail                       2351
horario                    2098
calle_nombre                  0
calle_altura                  0
calle_cruce                2817
direccion_completa            1
barrio                        0
comuna                        0
codigo_postal                57
codigo_postal_argentino      66
dtype: int64
------------------------------
Porcentaje de nulos
long                        0.0
lat                         0.0
id                          0.0
nombre                      0.0
categoria                   4.1
cocina                     12.7
ambientacion               19.8
telefono                   18.4
mail                       83.3
horario                    74.3
calle_nombre                0.0
calle_al

In [106]:
#DUPLICADOS

total_duplicados = df_original.drop(columns=['id']).duplicated().sum()
print(total_duplicados)

6


In [ ]:
#REPARACION DE ERRORES

df_original['ambientacion'] = df_original['ambientacion'].replace('CLASICO PORTE�O', 'CLASICO PORTEÑO')
df_original['ambientacion'] = df_original['ambientacion'].replace('ESPA�OL', 'ESPAÑOL')
df_original['ambientacion'] = df_original['ambientacion'].replace('DISE�O', 'DISEÑO')
df_original['ambientacion'] = df_original['ambientacion'].replace('HIST�RICA', 'HISTORICA')
df_original['calle_nombre'] = df_original['calle_nombre'].str.replace(r'HUMBERTO 1�', 'HUMBERTO PRIMO', regex=True)
df_original['horario'] = df_original['horario'].str.replace(r's�bado', 'sabado', regex=True)
df_original['horario'] = df_original['horario'].str.replace(r'mediod�a', 'mediodia', regex=True)
df_original['horario'] = df_original['horario'].str.replace(r'mi�rcoles', 'miercoles', regex=True)
df_original['horario'] = df_original['horario'].str.replace(r'd�as', 'dias', regex=True)
df_original['nombre'] = df_original['nombre'].replace('OTTO G�RGUELL', 'Otto Gorguel')
df_original['nombre'] = df_original['nombre'].replace('LA CIG�EÑA', 'LA CIGUEÑA')
df_original['nombre'] = df_original['nombre'].replace('Nuñez ', 'NUÑEZ')

In [ ]:
conteo = df_original['calle_nombre'].value_counts()

print(conteo)

print(f"TOTAL GENERAL DE REGISTROS: {conteo.sum()}")

calle_nombre
RIVADAVIA            109
CORRIENTES            78
CORDOBA               63
LAVALLE               61
SANTA FE              59
                    ... 
ECUADOR                1
SAN MARTIN AV          1
O'BRIEN                1
BUFANO ALFREDO R.      1
Camargo                1
Name: count, Length: 455, dtype: int64
TOTAL GENERAL DE REGISTROS: 2823


In [ ]:
conteo = df_original['ambientacion'].value_counts()

print(conteo)

print(f"TOTAL GENERAL DE REGISTROS: {conteo.sum()}")

ambientacion
CLASICO PORTEÑO         898
MODERNO                 875
ANTIGUO                  90
RUSTICA                  76
ITALIANA                 26
ESPAÑOL                  21
ELEGANTE                 21
DISEÑO                   18
FOLKLORICO               17
TANGO                    15
CHINA                    13
BOHEMIO                  13
AMERICANO                12
BODEGA                   11
BRITANICA                10
VANGUARDISTA             10
COLONIAL                 10
ORIENTAL                  8
MINIMALISTA               8
FUTBOL                    7
MEDITERRANEO              7
FRANCES                   6
MARINA                    6
MEXICANO                  6
CAMPESTRE                 5
ALEMANA                   5
ARABE                     4
POP                       4
PSICODELICO               4
TEATRAL                   4
ART- DECO                 4
ARMENIA                   4
AUTOS                     3
CINE                      3
GRIEGA                    3
ROMANTI

In [ ]:

errores_hora = df_original[df_original['horario'].str.contains('�', na=False)]

print(errores_hora['horario'])


16                             lunes a s�bado de 8 a 00
30                lunes a s�bado de 9 a 16 y de 19 a 21
42        lunes a viernes de 12 a 00 - s�bado de 20 a 3
54                             lunes a s�bado de 8 a 20
62                             lunes a s�bado de 8 a 22
                              ...                        
2791                           lunes a s�bado de 9 a 00
2792    lunes a viernes de 8.30 a 20 - s�bado de 9 a 00
2800    domingo a jueves de 20.30 a 00 - viernes y sï¿...
2802                   lunes a domingo mediod�a y noche
2807     lunes a s�bado 12 a 00 - domingo de 12 a 16.30
Name: horario, Length: 222, dtype: object


In [ ]:
errores_nombre = df_original[df_original['nombre'].str.contains('�', na=False)]

print(errores_nombre[['nombre','long','lat']])

                    nombre          long           lat
1387    LA CIG�EÑA   -58,4001076  -34,59674776
2080  OTTO G�RGUELL  -58,43646322  -34,58587824


In [ ]:
tipo_cocina = df_original['cocina'].value_counts()

print(tipo_cocina)

print(f'Cantidad: {tipo_cocina.sum()}')

cocina
MINUTAS             737
INTERNACIONAL       448
TRADICIONAL         297
PIZZERIA            176
PARRILLA            159
SANDWICHERIA        127
FAST FOOD            81
ITALIANA             77
MEDITERRANEA         48
PORTEÑA             39
DE AUTOR             32
REGIONAL             30
ESPAÑOLA            25
CHINA                17
SUSHI                12
TENEDOR LIBRE        11
JAPONESA             10
AMERICANA             9
TAPAS                 8
FUSION                8
FRANCESA              8
MEXICANA              7
NATURISTA             7
MARISCOS              7
MODERNO               7
ARMENIA               6
ARABE                 6
BOLIVIANA             6
CUBANA                6
GOURMET               5
PERUANA               4
ORIENTAL              4
VEGETARIANA           4
CLASICO PORTEÑO      4
BRASILERA             3
JUDIA                 3
PATAGONICA            3
PANQUEQUER�?A       3
ALEMANA               3
TAILANDESA            3
ASIATICA              2
CENTROEUR

In [ ]:
#LIMPIEZA DUPLICADOS

columnas = [col for col in df_original.columns if col != 'id']

df_limpio = df_original.drop_duplicates(subset=columnas, keep='first')

print(f"Filas originales: {len(df_original)}")
print(f"Filas después de limpiar: {len(df_limpio)}")

Filas originales: 2823
Filas después de limpiar: 2817


In [ ]:
#LIMPIEZA NULOS

df_original["cocina"] = df_original["cocina"].fillna("GASTRONOMIA GENERAL")
df_original["categoria"] = df_original["categoria"].fillna("OTROS")
df_original["ambientacion"] = df_original["ambientacion"].fillna("GENERAL")
df_original["telefono"] = df_original["telefono"].fillna("NO DISPONIBLE")
df_original["mail"] = df_original["mail"].fillna("N/A")
df_original["horario"] = df_original["horario"].fillna("NO ESPECIFICADA")
df_original["calle_cruce"] = df_original["calle_cruce"].fillna("N/A")
df_original["direccion_completa"] = df_original["direccion_completa"].fillna("UBICACIÓN POR COORDENADAS")

#CASO DE CP Y CPA


df_original['codigo_postal'] = df_original['codigo_postal'].astype(str).str.strip()
df_original['codigo_postal_argentino'] = df_original['codigo_postal_argentino'].astype(str).str.strip()

for col in ['codigo_postal', 'codigo_postal_argentino']:
    df_original[col] = df_original[col].replace(['nan', 'None', '', 'NaN'], None)
    

#TOMAMOS EL CPA PARA RELLENAR LOS CP CON SU VALOR

cp_extraido_del_cpa = df_original['codigo_postal_argentino'].str.extract(r'(\d{4})')[0]

df_original['codigo_postal'] = df_original['codigo_postal'].fillna(cp_extraido_del_cpa)

#TOMAMOS EL CP PARA LLENENAR EL CPA

cpa_estimado = "C" + df_original['codigo_postal'].astype(str) + "XXX"

df_original['codigo_postal_argentino'] = df_original['codigo_postal_argentino'].fillna(cpa_estimado)

#HACERMOS EL RELLENADO DE NULOS DE AMBOS

valores_relleno_final = {
    'codigo_postal': 'SIN CP',
    'codigo_postal_argentino': 'SIN CPA'
}
df_limpio = df_original.fillna(value=valores_relleno_final)


df_limpio['codigo_postal_argentino'] = df_limpio['codigo_postal_argentino'].replace('CNoneXXX', 'SIN CPA')

In [ ]:
#LIMPIEZA NULOS

df_original["cocina"] = df_original["cocina"].fillna("GASTRONOMIA GENERAL")
df_original["categoria"] = df_original["categoria"].fillna("OTROS")
df_original["ambientacion"] = df_original["ambientacion"].fillna("GENERAL")
df_original["telefono"] = df_original["telefono"].fillna("NO DISPONIBLE")
df_original["mail"] = df_original["mail"].fillna("N/A")
df_original["horario"] = df_original["horario"].fillna("NO ESPECIFICADA")
df_original["calle_cruce"] = df_original["calle_cruce"].fillna("N/A")
df_original["direccion_completa"] = df_original["direccion_completa"].fillna("UBICACIÓN POR COORDENADAS")

#CASO DE CP Y CPA


df_original['codigo_postal'] = df_original['codigo_postal'].astype(str).str.strip()
df_original['codigo_postal_argentino'] = df_original['codigo_postal_argentino'].astype(str).str.strip()

for col in ['codigo_postal', 'codigo_postal_argentino']:
    df_original[col] = df_original[col].replace(['nan', 'None', '', 'NaN'], None)
    

#TOMAMOS EL CPA PARA RELLENAR LOS CP CON SU VALOR

cp_extraido_del_cpa = df_original['codigo_postal_argentino'].str.extract(r'(\d{4})')[0]

df_original['codigo_postal'] = df_original['codigo_postal'].fillna(cp_extraido_del_cpa)

#TOMAMOS EL CP PARA LLENENAR EL CPA

cpa_estimado = "C" + df_original['codigo_postal'].astype(str) + "XXX"

df_original['codigo_postal_argentino'] = df_original['codigo_postal_argentino'].fillna(cpa_estimado)

#HACERMOS EL RELLENADO DE NULOS DE AMBOS

valores_relleno_final = {
    'codigo_postal': 'SIN CP',
    'codigo_postal_argentino': 'SIN CPA'
}
df_limpio = df_original.fillna(value=valores_relleno_final)


df_limpio['codigo_postal_argentino'] = df_limpio['codigo_postal_argentino'].replace('CNoneXXX', 'SIN CPA')

In [ ]:
#CAMBIO DE TIPO

df_original['long'] = df_original['long'].astype(str).str.replace(',', '.', regex=False).str.strip()
df_original['long'] = pd.to_numeric(df_original['long'], errors='coerce')

df_original['lat'] = df_original['lat'].astype(str).str.replace(',', '.', regex=False).str.strip()
df_original['lat'] = pd.to_numeric(df_original['lat'], errors='coerce')

In [22]:
df_original = pd.read_csv("oferta_gastronomica.csv", encoding="latin1", sep=";")
print(f" Archivo original cargado. Filas iniciales: {len(df_original)}")

# CREACION DE UNA COPIA DE ARCHIVO

df = df_original.copy()

# LIMPIEZA DE TEXTO Y ERRORES TIPOGRÁFICOS

df['ambientacion'] = df['ambientacion'].replace('CLASICO PORTE�O', 'CLASICO PORTENIO')
df['ambientacion'] = df['ambientacion'].replace('ESPA�OL', 'ESPANIOL')
df['ambientacion'] = df['ambientacion'].replace('DISE�O', 'DISENIO')
df['ambientacion'] = df['ambientacion'].replace('HIST�RICA', 'HISTORICA')
df['calle_nombre'] = df['calle_nombre'].str.replace(r'HUMBERTO 1�', 'HUMBERTO PRIMO', regex=True)
df['horario'] = df['horario'].str.replace(r's�bado', 'sabado', regex=True)
df['horario'] = df['horario'].str.replace(r'mediod�a', 'mediodia', regex=True)
df['horario'] = df['horario'].str.replace(r'mi�rcoles', 'miercoles', regex=True)
df['horario'] = df['horario'].str.replace(r'd�as', 'dias', regex=True)
df['nombre'] = df['nombre'].replace('OTTO G�RGUELL', 'Otto Gorguel')
df['nombre'] = df['nombre'].replace('LA CIG�EÑA', 'LA CIGUENIA')
df['cocina'] = df['cocina'].replace('PORTEÑA', 'PORTENIA')
df['cocina'] = df['cocina'].replace('PANQUEQUER�?A ', 'PANQUEQUERIA')
df['cocina'] = df['cocina'].replace('ESPAÑOLA', 'ESPANIOLA')
df['cocina'] = df['cocina'].replace('CLASICO PORTEÑO', 'CLASICO PORTENIO')
df['barrio'] = df['barrio'].replace('Nuñez', 'Nunez')

#CANTIDAD DE DUPLICADO

total_duplicados = df.drop(columns=['id']).duplicated().sum()
print(f"CANTIDAD DE DUPLICADOS ANTES DE LIMPIEZA:{total_duplicados}")

#LIMPIEZA DE DUPLICADOS

columnas = [col for col in df.columns if col != 'id']

df_limpio = df.drop_duplicates(subset=columnas, keep='first')

print(f"Filas originales: {len(df)}")
print(f"Filas después de limpiar: {len(df_limpio)}")

#CONVERSIÓN DE TIPOS DE DATOS INCORRECTOS

df['long'] = df['long'].astype(str).str.replace(',', '.', regex=False).str.strip()
df['long'] = pd.to_numeric(df['long'], errors='coerce')

df['lat'] = df['lat'].astype(str).str.replace(',', '.', regex=False).str.strip()
df['lat'] = pd.to_numeric(df['lat'], errors='coerce')

df['nombre'] = df['nombre'].astype(str)
df['codigo_postal'] = df['codigo_postal'].astype(str).str.strip()
df['codigo_postal_argentino'] = df['codigo_postal_argentino'].astype(str).str.strip()


#LIMPIEZA NULOS

df["cocina"] = df["cocina"].fillna("Gastronomia General")
df["categoria"] = df["categoria"].fillna("Otros")
df["ambientacion"] = df["ambientacion"].fillna("General")
df["telefono"] = df["telefono"].fillna("No disponible")
df["mail"] = df["mail"].fillna("No disponible")
df["horario"] = df["horario"].fillna("NO ESPECIFICADA")
df["calle_cruce"] = df["calle_cruce"].fillna("Es avenida")
df["direccion_completa"] = df["direccion_completa"].fillna("UBICACIÓN POR COORDENADAS")

#CASO DE CP Y CPA


df['codigo_postal'] = df['codigo_postal'].astype(str).str.strip()
df['codigo_postal_argentino'] = df['codigo_postal_argentino'].astype(str).str.strip()

for col in ['codigo_postal', 'codigo_postal_argentino']:
    df[col] = df[col].replace(['nan', 'None', '', 'NaN'], None)
    

#TOMAMOS EL CPA PARA RELLENAR LOS CP CON SU VALOR

cp_extraido_del_cpa = df['codigo_postal_argentino'].str.extract(r'(\d{4})')[0]

df['codigo_postal'] = df['codigo_postal'].fillna(cp_extraido_del_cpa)

#TOMAMOS EL CP PARA LLENENAR EL CPA

cpa_estimado = "C" + df['codigo_postal'].astype(str) + "XXX"

df['codigo_postal_argentino'] = df['codigo_postal_argentino'].fillna(cpa_estimado)

#HACERMOS EL RELLENADO DE NULOS DE AMBOS

valores_relleno_final = {
    'codigo_postal': 'SIN CP',
    'codigo_postal_argentino': 'SIN CPA'
}
df_limpio = df.fillna(value=valores_relleno_final)


df_limpio['codigo_postal_argentino'] = df_limpio['codigo_postal_argentino'].replace('CNoneXXX', 'SIN CPA')

df['codigo_postal'] = df['codigo_postal'].fillna('SIN CP')

OFERTA_GASTRONOMICA = "oferta_gastronomica_limpio.csv"
df.to_csv(OFERTA_GASTRONOMICA, index=False, encoding="latin1", sep=";")


print(f"Cantidad total de nulos en el nuevo archivo: {df.isnull().sum().sum()}")
print(f"Tipos clave: Longitud ({df['long'].dtype}), Latitud ({df['lat'].dtype})")



 Archivo original cargado. Filas iniciales: 2823
CANTIDAD DE DUPLICADOS ANTES DE LIMPIEZA:6
Filas originales: 2823
Filas después de limpiar: 2817
Cantidad total de nulos en el nuevo archivo: 0
Tipos clave: Longitud (float64), Latitud (float64)


In [23]:
df = pd.read_csv('oferta_gastronomica_limpio.csv', sep=';', encoding='latin-1', on_bad_lines='skip')


A partir de este aca se trabaja con el csv limpio

In [24]:
df.info()

<class 'pandas.core.frame.DataFrame'>
RangeIndex: 2823 entries, 0 to 2822
Data columns (total 18 columns):
 #   Column                   Non-Null Count  Dtype  
---  ------                   --------------  -----  
 0   long                     2823 non-null   float64
 1   lat                      2823 non-null   float64
 2   id                       2823 non-null   int64  
 3   nombre                   2823 non-null   object 
 4   categoria                2823 non-null   object 
 5   cocina                   2823 non-null   object 
 6   ambientacion             2823 non-null   object 
 7   telefono                 2823 non-null   object 
 8   mail                     2823 non-null   object 
 9   horario                  2823 non-null   object 
 10  calle_nombre             2823 non-null   object 
 11  calle_altura             2823 non-null   int64  
 12  calle_cruce              2823 non-null   object 
 13  direccion_completa       2823 non-null   object 
 14  barrio                  

In [27]:
df = pd.read_csv("oferta_gastronomica_limpio.csv", sep=';', encoding='latin-1', on_bad_lines='skip')

In [ ]:
#¿Qué categorías gastronómicas tienen mayor presencia?

mas_pedido = df.groupby(["cocina"])["cocina"].count().sort_values(ascending=False).head(5)


porcentaje_pedido = (
    mas_pedido / cantidad_total * 100
)

print("\nPorcentaje de gastronomia con mayor presencia:")

print(round(porcentaje_pedido, 2), "%")

cocina
MINUTAS                26.11
INTERNACIONAL          15.87
Gastronomia General    12.72
TRADICIONAL            10.52
PIZZERIA                6.23
Name: cocina, dtype: float64 %


In [ ]:
#¿Qué barrios tienen mayor presencia de propuestas saludables o alternativas?

saludables = [
    "NATURISTA",
    "VEGETARIANA"
]
saludables_df = df[
    df["cocina"].isin(saludables)
]

cantidad_total = len(df)
cantidad_saludables = len(saludables_df)

no_alternativo = ['Valor1', 'Valor2', 'Valor3']



print("\nCantidad de establecimientos saludables:")
print(cantidad_saludables)

print("\nDISTRIBUCIÓN POR BARRIO")
print(saludables_df["barrio"].value_counts())


Cantidad de establecimientos saludables:
11

DISTRIBUCIÓN POR BARRIO
barrio
Palermo        3
Recoleta       2
San Nicolas    2
Saavedra       1
Colegiales     1
Caballito      1
Floresta       1
Name: count, dtype: int64


In [57]:
#¿Existen zonas con potencial de crecimiento?

barrios_mayor_presencia = df.groupby(["barrio"])["barrio"].count().sort_values(ascending=False).head(5)

barrios_crecimiento = df.groupby(["barrio"])["barrio"].count().sort_values(ascending=False).tail(5)


print("\n Barrios con potencial de crecimiento:")
print(barrios_crecimiento)

print("\n Barrios con mayor presencia de establecimientos:")
print(barrios_mayor_presencia)


 Barrios con potencial de crecimiento:
barrio
Villa Santa Rita    4
Villa Luro          3
Agronomia           3
Paternal            2
Villa Soldati       2
Name: barrio, dtype: int64

 Barrios con mayor presencia de establecimientos:
barrio
Palermo        467
San Nicolas    454
Recoleta       327
Balvanera      215
Monserrat      149
Name: barrio, dtype: int64


CONCLUSIONES FINALES

Los tipos de cocina con mayor presencia en la ciudad de Buenos Aires son:

MINUTAS:                26.11%

INTERNACIONAL:          15.87%

Gastronomia General:    12.72%

TRADICIONAL:            10.52%

PIZZERIA:                6.23%


Esto nos indica que podemos crear opciones saludables y alternativas centrando en este tipo de cocina, ya que es la que más presencia tiene en la ciudad.


Los barrios con mayor presencia de establecimientos saludables son: PALERMO, RECOLETA, SAN NICOLAS, SAAVEDRA, COLEGIALES, CABALLITO Y FLORESTA
de estos barrios PALERMO, RECOLETA y SAN NICOLAS concentran la mayor cantidad de locales saludables.

Los barrios con potencial de crecimiento son: VILLA SOLDATI, PATERNAL, AGRONOMIA, VILLA LURO y VILLA SANTA RITA. 
Estos barrios tienen una menor presencia de establecimientos , 
lo que indica que podrían ser áreas con oportunidades para el desarrollo de propuestas gastronómicas alternativas y saludables.